hl/src/H5PT.c

```diff
@@ -29,7 +29,7 @@ static H5I_type_t H5PT_ptable_id_type = H5I_UNINIT;
 #define H5PT_HASH_TABLE_SIZE 64
 
 /* Packet Table private functions */
-static herr_t H5PT_free_id(void *id, void **_ctx);
+static herr_t H5PT_free_id(void *id);
```

Comment:
The ctx parameter was added to the public H5I_free_t typedef in 1.13 for async operations, but isn't in the 1.12 branch.